atoms/typography/_headings.scss

```diff
@@ -30,6 +30,12 @@ $heading-line-height-large: 2rem !default; // 32px
     line-height: $heading-line-height-large;
     font-weight: 300;
   }
+  // Medium
+  h3 {
+    font-size: $space-l;
```

Comment:
Do we still need the top var's ? Because we are now mixing `$space` values with header-font-size-x values ?

Is there a difference between `$heading-font-size-large` vs `$space-l`?

Reply:
The top vars are to be able to configure the design system for custom needs. I would introduce new top vars for the size and height :+1:

Reply:
I improved the variable naming